pyproject.toml

```diff
@@ -26,8 +26,14 @@ exclude = '(_notebooks/.*)'
 [tool.mypy]
 files = [
     "src/pytorch_lightning",
+    "src/lightning_app/cli/lightning_cli*.py",
+    "src/lightning_app/cli/cmd_*.py",
```

Comment:
can we rather ad testing all app folders and list excluded below in modules?
so it is consistent with pytorch typing check setting

Reply:
let me resolve the merge conflicts first and then change the pyproject.toml..

Reply:
resolved and followed your suggestions @Borda